js/timepicker.js

```diff
@@ -496,64 +496,59 @@
     }
 
     _inputFromTextField() {
-      var isHours = this.currentView === 'hours';
-      var unit = Math.PI / (isHours ? 6 : 30);
-      var radius, radian;
-
-      if (this.options.twelveHour) {
-        radius = this.options.outerRadius;
-      }
+      const isHours = this.currentView === 'hours';
 
       if (isHours) {
-        var value = this['inputHours'].value;
+        let value = this['inputHours'].value;
```

Comment:
Is value reassigned ? if not, prefer the use of const. Same comment for line 516

Reply:
Yes, it should be const